packages/react-app/src/components/Modals/FundModal.js

```diff
@@ -1,13 +1,13 @@
 import { Button, useToast } from '@1hive/1hive-ui';
+import { noop } from 'lodash-es';
 import PropTypes from 'prop-types';
 import React, { useState } from 'react';
 import { GiTwoCoins } from 'react-icons/gi';
-import QuestProvider from '../../providers/QuestProvider';
-import { emptyFunc } from '../../utils/class-util';
+import QuestProvider from '../../Services/QuestService';
 import AmountFieldInput from '../Shared/FieldInput/AmountFieldInput';
 import ModalBase from './ModalBase';
 
-export default function FundModal({ questAddress, onClose = emptyFunc }) {
+export default function FundModal({ questAddress, onClose = noop }) {
```

Comment:
whats noop?

Reply:
no operation function (place holder function that does not crash if called but does nothing)